litmus-portal/graphql-server/graph/model/modes_factory.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+func NewCloningInputFrom(chaosHub CreateChaosHubRequest) CloningInput {
+	return CloningInput{
+		ProjectID:     chaosHub.ProjectID,
+		RepoBranch:    chaosHub.RepoBranch,
+		RepoURL:       chaosHub.RepoURL,
+		HubName:       chaosHub.HubName,
+		IsPrivate:     chaosHub.IsPrivate,
+		UserName:      chaosHub.UserName,
+		Password:      chaosHub.Password,
+		AuthType:      chaosHub.AuthType,
+		Token:         chaosHub.Token,
+		SSHPrivateKey: chaosHub.SSHPrivateKey,
+	}
+}
```

Comment:
Might be a good idea to move this to the `pkg/chaoshub`, the `graph/model` has auto-generated code and can be over-written in the future by mistake. This function is also specific to `pkg/chaoshub` so it makes more sense to have it in there